client_plugin/drivers/vfile/kvstore/etcdops/etcdops.go

```diff
@@ -161,12 +180,45 @@ func NewKvStore(dockerOps *dockerops.DockerOps) *EtcdKVS {
 	return e
 }
 
+// rejoinEtcdCluster function is called when a node need to rejoin a ETCD cluster
+func (e *EtcdKVS) rejoinEtcdCluster() error {
+	nodeID := e.nodeID
+	nodeAddr := e.nodeAddr
+	lines := []string{
```

Comment:
It's pointing to a data-dir at /etcd-data. And if there are already previous ETCD information inside, it will try to join the previous ETCD cluster. Only if there is no data inside the assigned data-dir, it will create a new one.